Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -897,6 +992,22 @@ private void InstantiatePressButtonPrefab(Vector3 position, Quaternion rotation,
             Assert.IsNotNull(pressButtonCylinder, "Object 'Cylinder' could not be found under example object Model_PushButton.");
         }
 
+        /// <summary>
+        /// Instantiates HoloLens2PressableButton
+        /// </summary>
+        private void InstantiatePressableButtonHoloLens2(Vector3 position, Quaternion rotation, out Interactable interactable, out Transform interactableTransform)
```

Comment:
There's a lot of similarity here between `InstantiatePressableButtonHoloLens2Toggle`, `InstantiatePressButtonPrefab` and `InstantiatePressableButtonHoloLens2`. Perhaps we could unify them into a single method and pass `prefabPath` and `transformPath` as arguments.

Reply:
Something like
```suggestion
        private void InstatiateButton(Vector3 position, Quaternion rotation, string prefabPath, string transformPath, out Interactable interactable, out Transform interactableTransform)

```

Reply:
I'm happy either way though, just a suggestion

Reply:
Yeah it makes much more sense.